lib/events-native.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @fileoverview
+ * @author Box
+ */
+
+Box.Events = (function(){
+    'use strict';
+
+    return {
+        on: function(element, type, listener) {
+            element.addEventListener(type, listener);
```

Comment:
Yep.

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

Note: useCapture became optional only in more recent versions of the major browsers; for example, it was not optional prior to Firefox 6. You should provide this parameter for broadest compatibility.